test/e2e/parallel/spacerequest_test.go

```diff
@@ -279,90 +279,3 @@ func TestCreateSpaceRequest(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
-
-// Update scenarios are not supported as of now since:
-// - the only tier that works with this controller is `appstudio-env`,
-//   the controller tries to create a TokenRequest for the `namespace-manager` SA , and the only tier that has this SA is the `appstudio-env` tier
```

Comment:
we do support tiers without SA since https://github.com/codeready-toolchain/host-operator/pull/972 was implemented , so the "update space request tierName" sub test is something that could be enabled/implemented now.

Reply:
Hello, @mfrancisc! Thank you so much for your messages!

Regarding "update space request tierName", I think `TestCreateSpaceRequest` is already checking it [here](https://github.com/codeready-toolchain/toolchain-e2e/blob/master/test/e2e/parallel/spacerequest_test.go#L85-L88)

Reply:
> Regarding "update space request tierName", I think TestCreateSpaceRequest is already checking it [here](https://github.com/codeready-toolchain/toolchain-e2e/blob/master/test/e2e/parallel/spacerequest_test.go#L85-L88)

That test is verifying that an update on the SubSpace field is being reset by the SpaceRequest controller. Basically ensuring that the SubSpace is managed by the SpaceRequest, and the spacerequest_controller reconciles any changes to the SubSpace setting back the value from the SpaceRequest. I hope I was able to explain the intention of test. 

I don't think we have a test that covers the update of a Tier in the SpaceRequest, but as mentioned I'm not sure we need one or we should invest into this as a priority. 

In case we don't want to enable/update this subtest I think it's fine to delete it.

Reply:
@mfrancisc , my bad, sorry :sweat_smile: For some reason, my mind read UpdateSpaceRequest.

I think you are right! At least, I did not find it.